packages/core/src/lib/core/access-control.ts

```diff
@@ -101,10 +101,24 @@ export type ResolvedFieldAccessControl = {
 };
 
 export function parseListAccessControl(
-  access: ListAccessControl<BaseListTypeInfo> | undefined
+  access: ListAccessControl<BaseListTypeInfo>
 ): ResolvedListAccessControl {
   let item, filter, operation;
 
+  if (typeof access === 'function') {
+    return {
+      operation: {
+        create: access,
+        query: access,
+        update: access,
+        delete: access,
+      },
+
+      filter: { query: access, update: access, delete: access },
+      item: { create: access, update: access, delete: access },
```

Comment:
```suggestion
      filter: {
        create: () => true,
        update: () => true,
        delete: () => true
      },
      item: {
        create: () => true,
        update: () => true,
        delete: () => true
      },
```
Actually, I don't think we are defaulting these functions in this way, they are still optional as they are additive only